diff --git a/node_modules/@nuxtjs/vercel-builder/lib/index.js b/node_modules/@nuxtjs/vercel-builder/lib/index.js
index eed6178..c6b6152 100644
--- a/node_modules/@nuxtjs/vercel-builder/lib/index.js
+++ b/node_modules/@nuxtjs/vercel-builder/lib/index.js
@@ -322,7 +322,7 @@ async function build(opts) {
     "--prefer-offline",
     "--frozen-lockfile",
     "--non-interactive",
-    "--production=false",
+    "--include=dev",
     `--modules-folder=${modulesPath}`,
     `--cache-folder=${yarnCachePath}`
   ], { ...spawnOpts, env: { ...spawnOpts.env, NODE_ENV: "development" } }, meta);
